ko/django_urls/README.md

```diff
@@ -20,104 +20,79 @@ URL은 웹 주소랍니다. 웹 사이트를 방문할 때마다 브라우저의
 
 [...]
 """
-from django.conf.urls import url
 from django.contrib import admin
+from django.urls import path
 
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
+    path('admin/', admin.site.urls),
 ]
 ```
 
 장고가 이미 어떤 내용을 넣어 두었네요.
 
-`#`로 시작되는 줄은 주석이에요. 파이썬은 주석을 실행하지 않아요. 꽤 유용하겠죠?
+세 개의 따옴표들(`"""`, `'''`) 사이에 있는 줄들은 독스트링(docstring)이라 불려요. 파일의 최상단이나 클래스 혹은 메서드 위에 적어두면, 그것이 어떤 역할을 하는지 알려주는 역할을 합니다. 파이썬에서 실행되지는 않을 거예요.
```

Comment:
[Add Translation]
This explains about `docstring` 
Original Text: `Lines between triple quotes (''' or """) are called docstrings – you can write them at the top of a file, class or method to describe what it does. They won't be run by Python.`